src/components/edit-panel/edit-panel.css

```diff
@@ -0,0 +1,37 @@
+.edit-panel {
+    margin: 0 auto;
+    width: 600px;
+    margin-top: 50px;
+    color: black;
+}
+
+.buttons {
+    display: flex;
+    align-items: center;
+    justify-content: space-around;
+    color: #fff;
+}
+
+.btn-repeat {
+    background-color: rgb(134, 28, 28);
```

Comment:
try to use one format #FFF or rgb(255, 255, 255)
don't use RGBA(RGB)-color if you don't need opacity